src/order/heyting_algebra.lean

```diff
@@ -0,0 +1,136 @@
+/-
+Copyright (c) 2020 Chris A. Upshaw. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris A. Upshaw.
+
+Type class hierarchy for Heyting algebras.
```

Comment:
```suggestion
```
Some older files include descriptive lines like this in the copyright header, but we now prefer to omit them in favor of module doc strings.